stem/output.py

```diff
@@ -0,0 +1,255 @@
+import warnings
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from typing import List, Dict, Any
+from enum import Enum
+
+DEFAULT_GID_FLAGS = {
+    "WriteDeformedMeshFlag": "WriteUndeformed",
+    "WriteConditionsFlag": "WriteElementsOnly",
+    "GiDPostMode": "GiD_PostBinary",
```

Comment:
can you place these values in the KratosIO rather than output.py, as we dont expect the users to change this. (except GiDpostMode, because this might be handy, please place this in GiDOutputParameters then)